src/main/java/com/rarchives/ripme/ripper/rippers/EightmusesRipper.java

```diff
@@ -122,14 +123,10 @@ else if (!subUrl.startsWith("http://")) {
                     }
                     try {
                         logger.info("Retrieving full-size image location from " + parentHref);
-                        Thread.sleep(1000);
```

Comment:
I removed it because of your comment 

>  Behavior doesn't really make sense - delays between detecting each picture but doesn't rip anything until we get to the end, and then there is no delay. The delay as such will not help with rate limiting, and just arbitrarily slows down the download.

The sleep wasn't doing much to prevent anything as the image downloading was still done at full speed and in my testing (I ripped ~ 2 dozen comics) I never ran into any rate limiting or ip bans